AdminWebsite/AdminWebsite/ClientApp/src/app/booking/search-email/search-email.component.spec.ts

```diff
@@ -132,6 +138,31 @@ describe('SearchEmailComponent', () => {
         expect(component.results).toBeTruthy();
         expect(component.results.length).toEqual(0);
     });
+
+    it('should set errorNotFoundJohEmail to true when EJudFeature flag is ON and no results found from search service', () => {
+        featureFlagServiceSpy.getFeatureFlagByName.and.returnValue(of(true));
+        component.ngOnInit();
+        component.hearingRoleParticipant = 'Panel Member';
+        component.noDataFound();
+        expect(component.errorNotFoundJohEmail).toBeTruthy();
+    });
+
+    it('should set errorNotFoundJohEmail to false  when EJudFeature flag is OFF and no results found from search service', () => {
+        component.ngOnInit();
+        component.hearingRoleParticipant = 'Panel Member';
+        component.noDataFound();
+        expect(component.errorNotFoundJohEmail).toBeFalsy();
+    });
+
+    it('should show showCreateNewUserWarning when EJudFeature flag is OFF and no results found from search service', () => {
+        component.notFoundParticipant = true;
+
+        component.ngOnInit();
+        fixture.detectChanges();
+        component.hearingRoleParticipant = 'Panel Member';
+        expect(component.showCreateNewUserWarning).toBeTruthy();
```

Comment:
if showCreateNewUserWarning is a boolean we should use the toBe(true) assertion as toBeTruthy will pass if that property is simply defined